### PR TITLE
homepage fixes

### LINKED ISSUE #58: add account info on home page
- [x] link to my account in header
- [x] account name in navbar everywhere

### PR DESCRIPTION
**Pull request by isabellapedraza**
Added my account to header links on the homepage, and instead of adding the account name to the nav bar or header (it looked really clunky), I edited each page's title to use the account's name (and make the website more personalized, easier to understand it's not the logged in user's health records, but instead the account name's health records

### CODE REVIEW

File: client/components/HomePage.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <main>
     <section>
       <header>
-        <h1> The Future of Healthcare </h1>
+        <h1> Welcome @{{ $store.state.username }}  to The Future of Healthcare </h1>
```
Comment (jrdegreeff) [inline client/components/HomePage.vue]: should we have commas here if this is an appositive?

File: client/components/layout/NavBar.vue [hunk 2]
```diff
@@ -72,6 +71,7 @@ nav {
   white-space: nowrap;
 }
 
+
```
Comment (jrdegreeff) [inline client/components/layout/NavBar.vue]: why the extra line?

**Review by jrdegreeff — COMMENTED** [review]
just a couple minor comments.

As with the previous PR, please try to make your branches off of main to avoid all the commit duplication.

File: client/components/HomePage.vue (hunk 1 above)
Comment (wli-linda) [reply client/components/HomePage.vue]: "Welcome, @XXX, to the future of healthcare"?
Or maybe "Hi @XXX, welcome to the future of healthcare"?

Comment (isabellapedraza) [reply client/components/HomePage.vue]: I like a mix of both – "Welcome @xxx, to the future of healthcare"

Comment (jrdegreeff) [reply client/components/HomePage.vue]: Is that gramatically correct?

Comment (isabellapedraza) [reply client/components/HomePage.vue]: https://www.brandeis.edu/writing-program/resources/students/academic/style-grammar/comma-rules.html

I think it's ok according to rule 2

**Review by jrdegreeff — APPROVED** [review]

**Pull request merged**